Lib/fontTools/varLib/mutator.py

```diff
@@ -121,6 +122,35 @@ def instantiateVariableFont(varfont, location, inplace=False):
 		log.info("Building interpolated tables")
 		merger.instantiate()
 
+	for glyph in glyf.glyphs.values():
+		if hasattr(glyph, "program"):
+			instructions = glyph.program.getAssembly()
+			# If GETVARIATION opcode is used in bytecode of any glyph add IDEF
+			addidef = False
+			for instruction in instructions:
+				if instruction.startswith("GETVARIATION"):
+					addidef = True
+					break
+			if addidef:
+				asm = []
+				if varfont.has_key('fpgm'):
+					fpgm = varfont['fpgm']
+					asm = fpgm.program.getAssembly()
+				else:
+					fpgm = newTable('fpgm')
+					fpgm.program = ttProgram.Program()
+					varfont['fpgm'] = fpgm
+				log.info("Adding IDEF to fpgm table for GETVARIATION opcode")
+				asm.append("PUSHB[000] 145")
+				asm.append("IDEF[ ]")
+				args = [str(len(loc))]
+				for val in loc.values():
+					args.append(str(floatToFixed(val, 14)))
+				asm.append("NPUSHW[ ] " + ' '.join(args))
```

Comment:
Isn't it easier to produce the program skipping the string parsing stage?

Reply:
Sorry I'm not sure if I completely understood your comment. Can you explain :)

Reply:
Ignore me.  I thought the program data types is more than just strings.